admin-home.php

```diff
@@ -82,7 +82,8 @@
     'brightspace_url' => $tool['brightspace_url'],
     'brightspace_log_url' => $tool['brightspace_log_url'],
     'vula_url' => $tool['vula_url'],
-    'jira_url' => $tool['jira_url']
+    'jira_url' => $tool['jira_url'],
+    'conversion_test' => $LAUNCH->ltiRawParameter('conversion_test', false)
```

Comment:
In order for the 'Test conversion' checkbox to be checked on default, add "custom_" in front of the 'conversion_test' ltiRawParameter. 

So this line should be:

`'conversion_test' => $LAUNCH->ltiRawParameter('custom_conversion_test', false)`